README.md

```diff
@@ -171,7 +171,17 @@ listen to configure the `SPLUNK_DEBUG_CONFIG_SERVER_PORT` environment variable.
 
 You can use the environment variable `SPLUNK_LISTEN_INTERFACE` and associated installer option to configure the network
 interface on which the collector's receivers and telemetry endpoints will listen.
-The default value of `SPLUNK_LISTEN_INTERFACE` is set to `0.0.0.0`.
+The default value of `SPLUNK_LISTEN_INTERFACE` is set to `0.0.0.0`, except on Chocolatey and
+the Windows and Linux standalone installers, where it defaults to `127.0.0.1`.
+
+The table below shows the configuration switches to use at installation time:
+
+| Method     | Command                                                                             |
+|------------|-------------------------------------------------------------------------------------|
+| Chocolatey | `choco install splunk-otel-collector --params="'/SPLUNK_LISTEN_INTERFACE:0.0.0.0'"` |
+| Linux      | `sudo install.sh  --listen-interface=0.0.0.0`                                       |
```

Comment:
For consistency with the linux install command in our other docs, for example https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-installer.md
```suggestion
| Linux      | `curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && sudo sh /tmp/splunk-otel-collector.sh --realm SPLUNK_REALM -- SPLUNK_ACCESS_TOKEN --listen-interface 0.0.0.0`                                       |
```